lib/garage/controller_helper.rb

```diff
@@ -34,7 +34,7 @@ def resource_owner_id
     # Use this method to render 'unauthorized'.
     # Garage user may overwrite this method to response custom unauthorized response.
     # @return [Hash]
-    def unauthorized_render_options
+    def unauthorized_render_options(error: nil)
```

Comment:
Ah, my mistake, defining default value for keyword arguments accepts calling without any parameters.

```
[1] pry(main)> def f(err: nil); p err end
=> :f
[2] pry(main)> f
nil
=> nil
[3] pry(main)> f(1)
ArgumentError: wrong number of arguments (1 for 0)
from (pry):1:in `f'
[4] pry(main)> f(err: 1)
1
=> 1
```

>I think it could better to follow the upstream than using * as an argument

+1 for this. So I merge this patch.

I think it is not good idea to force users to write wrapper.